Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,329 @@
+---
+title: Ceph Dashboard Monitoring
+weight: 2500
+indent: true
+---
+{% include_relative branch.liquid %}
+
+## Ceph Dashboard Monitoring
+Ceph clusters deployed using the Rook operator do not provide by default the software pieces needed to allow the Ceph Dashboard provide the monitoring and alerting functionalities. This document together with several resources files explains how to install and configure these pieces and the Ceph dashboard in order to provide the monitoring and alerting functionality. 
+
+In first place is good to have a brief summary about what are the monitoring stack components we will need to install and configure.
+
+1. [Prometheus operator](#prometheus-operator): it records real-time metrics in a time series database
+2. [Prometheus Ceph manager module](#prometheus-ceph-manager-module): provides Ceph cluster metrics to Prometheus
+3. [Grafana](#grafana): it will allow us to represent graphically the information collected by Prometheus
+4. [Prometheus Node exporter](#prometheus-node-exporter): provides cluster nodes machine metrics to Prometheus
+5. [Prometheus alert manager](#prometheus-alert-manager): manage alerts produced in Prometheus
+
+
+## Prerequisites
+- A working Ceph cluster deployed using the Rook operator.
+- [Prometheus manager module must be enabled](https://docs.ceph.com/en/latest/mgr/prometheus/#enabling-prometheus-output)
+- [Dashboard must be enabled](ceph-dashboard.md#enable-the-ceph-dashboard) in the Ceph cluster CRD
+- A PV with available space in order to store Grafana database (500Mb)
+- The files with the different resources definition and configuration. You can get them from the Rook repository:
+```console
+git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+## Prometheus Operator
+Follow the instructions provided in [Prometheus Monitoring](https://github.com/rook/rook/blob/{{ branchName }}/Documentation/ceph-monitoring.md)
+
+The Prometheus operator allows you to set up a Prometheus instance in you kubernetes cluster, a summary of the commands you will need to execute is below:
+
+Install the Prometheus Operator
+```console
+kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.40.0/bundle.yaml
+```
+Go to the folder where the different resource files for the monitoring stack are stored
+```console
+cd rook/cluster/examples/kubernetes/ceph/monitoring
+```
+Create and configure the Prometheus instance 
```

Comment:
These steps are duplicated from ceph-monitoring.md? How can we consolidate?

Reply:
I wanted to provide a brief summary of the minimum things to do .. and let the user to get all the stuff from the same document without "jumping". I will clarify that with a previous comment.

Reply:
What if we refer them to the other document as a prerequisite? This still seems like extra duplication.